### PR TITLE
Fixes issue #548Added fossasia project link to navbar

### PR DESCRIPTION
**Pull request by arjunpandey2003**
Fixes issue #548 
Updated index.html added an id fossasia-projects(to refer to in head.html)
Updated header.html
Deploy link:
https://arjunpandey2003.github.io/gci17.fossasiapc.org/

### CODE REVIEW

**Review by nzec — APPROVED** [review]
Squash!

**Comment by arjunpandey2003** [comment]
yeah doing that

**Review by saarthakchats — CHANGES_REQUESTED** [review]
Why did you remove the dropdown - "Quick links" from the navbar?

**Comment by nzec** [comment]
@saarthakchats He did not.

**Comment by arjunpandey2003** [comment]
@saarthakchats i didnt

**Comment by arjunpandey2003** [comment]
@saarthakchats btw its still there

**Comment by saarthakchats** [comment]
It doesnt show on the deploy link

**Comment by saarthakchats** [comment]
aaaaaaaa
Where is it?

**Comment by arjunpandey2003** [comment]
it does
https://user-images.githubusercontent.com/25302509/34390657-274898d4-eb67-11e7-86de-58fe220c36ee.png

**Comment by arjunpandey2003** [comment]
check link this
and check the link its the deploy link

**Comment by saarthakchats** [comment]
It's not showing on my laptop. Never mind

**Comment by arjunpandey2003** [comment]
@saarthakchats thanks try again

**Comment by saarthakchats** [comment]
aaaa
This is what shows on my screen

**Comment by saarthakchats** [comment]
Forget it

File: index.html [hunk 1]
```diff
@@ -346,7 +347,8 @@ <h2>Open Event Organizer App</h2>
         </div>
         
     </div>
-</section>
+ </section>
```
Comment (0x48piraj) [inline index.html]: Fix it.

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
See code review :+1:

**Comment by arjunpandey2003** [comment]
@0x48piraj what do you want me to fix please elaborate

File: index.html [hunk 2]
```diff
@@ -295,7 +296,7 @@ <h6 class="uppercase mb16">Technologies used: Java, Shell Script, Python, Javasc
     </div>
     
 </section>
-<section class="image-edge">
+ <section class="image-edge">
```
Comment (ParthS007) [inline index.html]: @arjunpandey2003  Please remove whitespace

File: index.html (hunk 1 above)
Comment (ParthS007) [reply index.html]: @arjunpandey2003  Why you added the end `</section>`, It is already there. Please remove it

**Review by ParthS007 — CHANGES_REQUESTED** [review]

**Review by isuruAb — COMMENTED** [review]
quick links drop down does not visible in small screens

**Comment by arjunpandey2003** [comment]
@ParthS007 @0x48piraj done

**Review by ParthS007 — CHANGES_REQUESTED** [review]
@arjunpandey2003 whitespaces are still present, please remove them

**Comment by ParthS007** [comment]
@arjunpandey2003  Now you have included previous commits in your PR 😟 .Please check or make a new PR by closing this 👍

**Pull request closed without merging**